spec.bs

```diff
@@ -849,6 +857,17 @@ a reporting destination=] through the reference it kept, it will handle all of t
 stored in the [=list=]. If the destination is never finalized, then the pending events will never
 be sent.
 
+An <dfn for=fencedframetype>untrusted network status</dfn> is either "<dfn export for='untrusted
+network status'>`enabled`</dfn>", "<dfn export for='untrusted network status'>`disabled for this
+tree`</dfn>", or "<dfn export for='untrusted network status'>`disabled for all subtrees`</dfn>".
+
+Note: [=untrusted network status/Disabled for this tree=] is not the final network cutoff state. It
+is an intermediate state where the frame has had its network access revoked, but at least one
+subframe still has network access. It does not get special API access at this stage as any
```

Comment:
When you say "but at least one subframe still has network access" do you mean normal child subframe, or is this only limited to sub-fenced-frame trees? I think you mean the latter, so I'd clarify that everything inside of the current tree (restricted to fences) suddenly gets network cut-off immediately.